docs/configuration/thumbnails.md

```diff
@@ -62,22 +62,42 @@ Here is a little example with a `teaser` and a `cover` alias:
 thumbnails:
     aliases:
         teaser:
-            size: [400,300]
+            size: [400, 300]
             cropping: crop
         cover:
-            size: [600,400]
+            size: [600, 400]
             cropping: crop
 ```
 
-As you can see, each alias has a new setting called `size` where you define
-the width and the height of the thumbnails. You can also define how they will be cropped.
+As you can see, each alias has a new setting called `size` where you define the
+width and the height of the thumbnails. You can also define how they will be
+cropped.
+
+Basic usage is straightforward:
+
+```twig
+{{ record.image|thumbnail(400, 300, 'c') }}
+
+becomes:
+
+{{ record.image|thumbnail('teaser') }}
+```
+
+For further examples on how to use the thumbnail alias with your code, please
+see the [template tag documentation][].
 
 ### Preventing non-alias Thumbnails
 
-With the `only_aliases` setting in the general thumbnail configuration,
-you can prevent the generation of thumbnails that don't belong to an alias.
+With the `only_aliases` setting in the general thumbnail configuration, you can
+prevent the generation of thumbnails that don't belong to an alias. This is
+useful to secure your server from automatically generated thumbnails that could
+flood your cache.
+
+When developing a site with this feature turned on, there are two things to note:
 
-This is useful to secure your server from automatically
-generated thumbnails that could flood your cache.
+ - If you're logged on, thumbnails not using an alias will still work. Test them
+   'incognito', or when logged off.
+ - Images that are already generated might still be served from cache. Flush the
+   cache, if you're seeing stale images.
 
-For examples on how to use the thumbnail alias with your code, please see the [template tag documentation](https://docs.bolt.cm/3.2/templating/templatetags#thumbnail).
+[template tag documentation]: https://docs.bolt.cm/3.2/templating/templatetags#thumbnail
```

Comment:
I know this is a bit of a hard one, but shouldn't we avoid a version here? The link won't appear before 3.2 docs anyway and it would save the need to update on each version branch